other/materials_designer/create_slab.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>Update the header to match other notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>